plugin-server/src/worker/ingestion/event-pipeline/populateTeamDataStep.ts

```diff
@@ -33,6 +74,5 @@ export async function populateTeamDataStep(
         ip: team.anonymize_ips ? null : event.ip,
     }
 
-    delete event['token']
```

Comment:
keeping it as it can be useful through the new kafka partition statistics table.